weaviate/collection/classes.py

```diff
@@ -324,9 +324,30 @@ class _MultiTenancyConfig:
     enabled: bool
 
 
+@dataclass
+class _ReferenceDataType:
+    collections: List[str]
+
+
+class ReferenceDataType(BaseModel):
```

Comment:
Do you think the UX of
```
ReferenceProperty(name="ref", reference=ReferenceDataType(collections=["A", "B"])),
ReferenceProperty(name="ref", reference=ReferenceDataType(collections="A")),
```
is better than

```
ReferenceProperty(name="ref", reference=["A", "B"]),
ReferenceProperty(name="ref", reference="A"),
```

?

Reply:
No now that it's spelt out 😁 I introduced `ReferenceDataType` as a Pydantic input model to complement the `_ReferenceDataType` dataclass return model but the nested classes here is cumbersome

Reply:
Perhaps doing the same here with `LinkTo` and `ReferenceTo` by having `ReferenceProperty` and `ReferencePropertyMultiTarget`?